workflow-templates/crowdin-download.yml

```diff
@@ -0,0 +1,55 @@
+name: Crowdin download
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
```

Comment:
theoretically, would it be useful to allow the consumer to be able to change the cron schedule as well? Don't know if that's possible here & if we can still default to this value if not provided